state/lease/client_race_test.go

```diff
@@ -310,21 +310,14 @@ var _ = gc.Suite(&ClientNTPSuite{})
 
 func (s *ClientNTPSuite) TestTimeGoesForwards(c *gc.C) {
 	f := s.EasyFixture(c)
-	f.Clock.step = 2 * time.Millisecond
+	f.Clock.step = 200 * time.Millisecond
 	err := f.Client.Refresh()
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-func (s *ClientNTPSuite) TestTimeGoesBackwardsALittle(c *gc.C) {
+func (s *ClientNTPSuite) TestTimeGoesBackwards(c *gc.C) {
```

Comment:
is this one of the tests you have to fix? because I don't think it makes sense anymore. we're only using the wall clock for the initial time stamp, so the step doesn't even come into play